learning_observer/learning_observer/auth/handlers.py

```diff
@@ -55,8 +55,11 @@ async def user_from_session(request):
     '''
     session = await aiohttp_session.get_session(request)
     session_user = session.get(constants.USER, None)
-    if constants.AUTH_HEADERS in session:
-        request[constants.AUTH_HEADERS] = session[constants.AUTH_HEADERS]
+    header_keys = [constants.AUTH_HEADERS, constants.CANVAS_AUTH_HEADERS]
```

Comment:
Code like this won't scale easily to more LMSs. We ought to have a better auth system that can handle multiple auths at once.
Example: a teacher might need their roster from Canvas, but also needs to sign into Google to get updated document text.